docs/userguide/networking/index.md

```diff
@@ -1,21 +1,541 @@
 <!--[metadata]>
 +++
-title = "Network configuration"
-description = "Docker networking feature is introduced"
-keywords = ["network, networking, bridge, docker,  documentation"]
+aliases=[
+"/engine/userguide/networking/dockernetworks"
+]
+title = "Docker container networking"
+description = "How do we connect docker containers within and across hosts ?"
+keywords = ["Examples, Usage, network, docker, documentation, user guide, multihost, cluster"]
 [menu.main]
-identifier="smn_networking"
-parent= "engine_guide"
-weight=7
+parent = "smn_networking"
+weight = -5
 +++
 <![end-metadata]-->
 
-# Docker networks feature overview
+# Understand Docker container networks
 
-This sections explains how to use the Docker networks feature. This feature allows users to define their own networks and connect containers to them. Using this feature you can create a network on a single host or a network that spans across multiple hosts.
+To build web applications that act in concert but do so securely, use the Docker
+networks feature. Networks, by definition, provide complete isolation for
```

Comment:
@diogomonica , these are the legacy networking docs that need an update. question on this PR pertains mostly to `overlay-security-model.md`